### PR TITLE
RadViz quick methods updates

### PR DESCRIPTION
**Pull request by naresh-bachwani**
Addressing issue #960
I have made the following changes:

1. Added show kwarg to RadViz.
2. Added test for the quick method
3. Updated documentation

### Sample Code and Plot
```python
    from yellowbrick.features.radviz import radviz
    from yellowbrick.datasets import load_occupancy
    
    #Load the classification dataset
    X, y = load_occupancy()

    # Specify the target classes
    classes = ["unoccupied", "occupied"]

    # Instantiate the visualizer
    radviz(X, y, classes=classes)
```

### CHECKLIST

- [x] _Is the commit message formatted correctly?_
- [ ] _Have you noted the new functionality/bugfix in the release notes of the next release?_

- [ ] _Included a sample plot to visually illustrate your changes?_
- [x] _Do all of your functions and methods have docstrings?_
- [x] _Have you added/updated unit tests where appropriate?_
- [x] _Have you updated the baseline images if necessary?_
- [x] _Have you run the unit tests using `pytest`?_
- [x] _Is your code style correct (are you using PEP8, pyflakes)?_
- [x] _Have you documented your new feature/functionality in the docs?_

- [ ] _Have you built the docs using `make html`?_

### CODE REVIEW

File: docs/api/features/_radviz.rst.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
+
```
Comment (pdamodaran) [inline docs/api/features/_radviz.rst.html]: Please remove this file.

File: docs/api/features/radviz.rst [hunk 2]
```diff
@@ -22,6 +22,13 @@ percent missing.
 If you do receive this warning, you may want to look at imputation strategies.
 A good starting place is the `scikit-learn Imputer. <http://scikit-learn.org/stable/modules/generated/sklearn.impute.SimpleImputer.html>`_
 
+=================   =================
+Visualizer           `RadViz <https://www.scikit-yb.org/en/latest/api/features/radviz.html#yellowbrick.features.radviz.RadialVisualizer>`_
+Quick Method         `radviz() <https://www.scikit-yb.org/en/latest/api/features/radviz.html#yellowbrick.features.radviz.radviz>`_
+Models               Classification, Regression
+Workflow             Feature Analysis
```
Comment (pdamodaran) [inline docs/api/features/radviz.rst]: Please change this to Feature Engineering

File: docs/api/features/radviz.rst [hunk 3]
```diff
@@ -45,10 +52,32 @@ A good starting place is the `scikit-learn Imputer. <http://scikit-learn.org/sta
 For regression, the ``RadViz`` visualizer should use a color sequence to
 display the target information, as opposed to discrete colors.
 
+
+Quick Method
+-------------------------
+The same functionality above can be achieved with the associated quick method ``radviz``. This method will build the ``RadViz`` object with the associated arguments, fit it, then (optionally) immediately show it
```
Comment (pdamodaran) [inline docs/api/features/radviz.rst]: Can you change the above to:
The same functionality above can be achieved with the associated quick method ``radviz``. This method will build the ``RadViz`` object with the associated arguments, fit it, then (optionally) immediately show the visualization.

File: yellowbrick/features/radviz.py [hunk 4]
```diff
@@ -77,6 +78,11 @@ def radviz(
         Specify a transparency where 1 is completely opaque and 0 is completely
         transparent. This property makes densely clustered points more visible.
 
+    show: bool, default: True
+        If True, calls ``show()``, which in turn calls ``plt.show()`` however you cannot
+        call ``plt.savefig`` from this signature, nor ``clear_figure``. If False, simply
+        calls ``finalize()``
+
```
Comment (pdamodaran) [inline yellowbrick/features/radviz.py]: Could you add the description for `kwargs` below `show`

File: docs/api/features/_radviz.rst.html (hunk 1 above)
Comment (naresh-bachwani) [reply docs/api/features/_radviz.rst.html]: Sorry, my bad!

**Comment by naresh-bachwani** [comment]
@pdamodaran - thanks for reviewing the PR. I have made the suggested changes, except one. I don't think that we need to add the description for X and y to the parameter description for the Visualizer. `X` and `y` are not the parameters to the visualizer, but instead to fit and transform.
Just let me know if you still want to go ahead or I misinterpreted the comment.

**Comment by pdamodaran** [comment]
Thanks for making all of the changes @naresh-bachwani!  

Regarding the comment about `X` and `y` - I apologize for the confusion.  What I meant was if you look at the `Parameters` list in line 41 of the `radviz.py` code, the first parameter in the docstring is `ax`.  What I had asked is if you could copy the documentation for the `X` and `y` parameters that are in the quick method (lines 329 - 333) and paste them on line 43 under `Parameters` so that we are consistent.  

Let me know if you have any additional questions.  Thanks again for all of your work on this!

**Pull request merged**